science/proxmark3-iceman/Portfile

```
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                1.0
+PortGroup                 github 1.0
+
+name                      proxmark3-iceman
+version                   4.13441
+revision                  0
+license                   GPL-2
+categories                science comms
+platforms                 darwin macosx
```

"darwin" implies "macosx". ("darwin" means both "macosx" and "puredarwin".)
```suggestion
platforms                 darwin
```